src/test/plugin-configs/changelog/dont-display-file-and-rev-info-plugin-config.xml

```diff
@@ -0,0 +1,44 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-changelog-plugin</artifactId>
+                <configuration>
+                    <displayFileAndRevInfo>false</displayFileAndRevInfo>
+                    <type>range</type>
+                    <range>-1</range>
+                    <dateFormat>yyyy-MM-dd</dateFormat>
+                    <basedir>${basedir}/src/main/java</basedir>
+                    <outputXML>${basedir}/src/test/changelog-xml/min-changelog.xml</outputXML>
+                    <outputXMLExpiration>60</outputXMLExpiration>
+                    <outputEncoding>ISO-8859-1</outputEncoding>
```

Comment:
UTF-8?

Reply:
I can't change it to UTF-8 unless I change all IT tests to UTF-8. I don't no why but it turns out the the xml encoding of the `min-changelog.xml` file has influence. This file is used in common, so that's why all tests have to change then. Please let me know what you think is wise at this stage